applications/launchpad/gui-react/src/components/CoinsList/index.tsx

```diff
@@ -1,9 +1,20 @@
 import Loading from '../Loading'
 import Text from '../Text'
 
-import { CoinsListItem, StyledCoinsList } from './styles'
+import { CoinsListItem, IconWrapper, StyledCoinsList } from './styles'
 import { CoinsListProps } from './types'
 
+const formatAmount = (amount: string) => {
+  if (Number(amount) === 0) {
+    return '00 000'
+  } else {
+    // Add spaces to number
```

Comment:
I'm just thinking, maybe instead of forcing this formatting we should use `toLocaleString()` ?

Reply:
I also thought about this, but it will use `,` instead of space, ie: `123,456.87`